src/test/OpenEXRTest/testPreviewImage.cpp

```diff
@@ -110,8 +110,8 @@ readWriteFiles (
 
         assert (dw == file2.dataWindow ());
 
-        int w  = dw.max.x - dw.min.x + 1;
```

Comment:
Here, int was pedantically correct, since datawindow is a box2i, which is a box<vec2i> and vec2i is a Vec2<int>... It probably makes sense to change the downstream size_t comparison warning instead?

Reply:
Done. This is reminiscent of some issues uncovered by the fuzzer where the data window is so large that the width calculation overflows an int, but it's obviously never occurred in this test, so fine to leave it as is.